packages/discussions/src/channels/types.ts

```diff
@@ -13,10 +13,12 @@ import {
   Role,
   SharingAccess,
 } from "../shared/types";
+import { IPost } from "../types";
```

Comment:
I think it might be safer to import this directly from  `../posts/types` in order to protect against a circular reference.

Reply:
👍